Added descriptions in readme for Global Settings Parameters. (#63)
MalteZA/MAES: README.md

|:---------------------------------------------|:-------|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| Logic Tick Delta Millis                      | Int    | The amount of simulated milliseconds between each logic tick, i.e. running of code from an exploration algorithm                                                                                                                                                                                                                                                                                                                                                    |

> | Physics Ticks Per Logic Update               | Int    | The amount of times Unity will continue to simulate physics between each logic update. Setting this to 1 will ensure, that no movement occurs between two logic ticks                                                                                                                                                                                                                                                                                               |

Reviewer: ```suggestion
| Physics Ticks Per Logic Update               | Int    | The amount of times Unity will continue to simulate physics between each logic update. Setting this to 1 will ensure that no movement occurs between two logic ticks                                                                                                                                                                                                                                                                                               |
```

> | Draw Communication                           | Bool   | If enabled uses Unity Gizmos to draw lines between communicating agents when the communicate (Can only be seen inside Unity Editor with Gizmos enabled)                                                                                                                                                                                                                                                                                                             |

Reviewer: ```suggestion
| Draw Communication                           | Bool   | If enabled, use Unity Gizmos to draw lines between communicating agents when the communicate (Can only be seen inside Unity Editor with Gizmos enabled)                                                                                                                                                                                                                                                                                                             |
```

> | Show Environment Tags                        | Bool   | If enabled draws the environment tags as Gizmos                                                                                                        (Can only be seen inside Unity Editor with Gizmos enabled)                                                                                                                                                                                                                                                   |

Reviewer: ```suggestion
| Show Environment Tags                        | Bool   | If enabled, draw the environment tags as Gizmos                                                                                                        (Can only be seen inside Unity Editor with Gizmos enabled)                                                                                                                                                                                                                                                   |
```

> | Should Write CSV Results                     | Bool   | If enabled statistics are written to CSV files whenever a simulation is finishes (Not when  the simulator is closed manually)                                                                                                                                                                                                                                                                                                                                       |

Reviewer: ```suggestion
| Should Write CSV Results                     | Bool   | If enabled, write statistics to a CSV files whenever a simulation is finished (Not when  the simulator is closed manually)                                                                                                                                                                                                                                                                                                                                       |
```

| Statistics Output Path                       | string | The directory where the CSV results are saved upon completing a simulation. Remember to end string with Path.DirectorySeparatorChar                                                                                                                                                                                                                                                                                                                                 |
| Ticks Per Stats Snapshot                     | Int    | The frequency of data collection from the simulation. If this value is 1 the state of the simulation is measured and saved at every logic tick                                                                                                                                                                                                                                                                                                                      |

> | Populate Adjacency And Com Groups Every Tick | Bool   | The adjacency matrix used inside the [CommunicationManager.cs](Assets/Scripts/Robot/CommunicationManager.cs) is populated lazily. Enabling this setting will make it eager. This can be useful for gathering statistics regarding communication ranges to test, if agents are at any time outside communication range, as opposed to testing only when communication actually occurs. Enabling this does, however, decrease performance, sometimes significantly so |

Reviewer: ```suggestion
| Populate Adjacency And Com Groups Every Tick | Bool   | The adjacency matrix used inside the [CommunicationManager.cs](Assets/Scripts/Robot/CommunicationManager.cs) is populated lazily. Enabling this setting will make it eager. This can be useful for gathering statistics regarding communication ranges to test if agents are at any time outside communication range, as opposed to testing only when communication actually occurs. Enabling this does, however, decrease performance - sometimes significantly so |
```
